server.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const net = require('net');
+const EE = require('events');
+const Client = require('./model/client.js');
+const PORT = process.env.PORT || 3000;
+const server = net.createServer();
+const ee = new EE();
+
+const pool = [];
+
+ee.on('default', function(client, string){
+  client.socket.write('not a command\n');
+});
+
+ee.on('@dm', function(client, string){
+  let message = string.split(' ').slice(1).join(' ').trim();
+  let callByName = string.split(' ').shift().trim();
+  pool.forEach(c => {
+    console.log('nickname:', c.nickname);
+    console.log('id:', c.id);
+    if(c.nickname === callByName || c.id === callByName){
+      c.socket.write(`${client.nickname} says: ${message}`);
```

Comment:
You should add a line return here after the message so their cursor will drop down to the next line.